client/components/Landing/LandingStats/LandingStats.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  faCalendarCheck, faHourglassHalf, faQuestion, faUserAstronaut, faUserNinja,
+} from '@fortawesome/free-solid-svg-icons';
+
+import { faSlideshare } from '@fortawesome/free-brands-svg-icons';
+import CounterCard from './CounterCard/CounterCard';
+
+import styles from './LandingStats.module.scss';
+
+interface Props {
+  participants: number;
+  events: number;
+  trainers: number;
+  hoursOfTraining: number;
+  slides: number;
+  countFrontEndWord: number;
+}
+
+export default function LandingStats({
+  participants, events, trainers, hoursOfTraining, slides, countFrontEndWord,
+} : Props) {
+  return (
+    <div className={`${styles['landing-stats']}`}>
+      <section>
+        <h2 className="text-center">
+          Statistici
+        </h2>
+        <ul className="d-flex justify-content-center align-items-center">
+          <li>
```

Comment:
We should have an array passed as a prop with all this info, and here we should do a `.map` over it.

This way:
1. We get rid of manually having to insert things here (ie: duplicate code)
2. We can re-use this component in other pages, but with different stats